recipes/correct-ts-specifiers/src/workflow.ts

```diff
@@ -0,0 +1,60 @@
+import module from 'node:module';
+
+import { type Api, api } from '@codemod.com/workflow';
+import type { Helpers } from '@codemod.com/workflow/dist/jsFam.d.ts';
+
+import { mapImports } from './map-imports.ts';
+import type { FSAbsolutePath } from './index.d.ts';
+
+module.register('@nodejs-loaders/alias', import.meta.url);
```

Comment:
you can just import register because it's the only api we use on this file 

```suggestion
register('@nodejs-loaders/alias', import.meta.url);
```

Reply:
I thought of that. I went with `module.register` to make it more clear what is happening (lots of APIs have `register`).

Is there a perf improvement to this or is it just 4 characters fewer?

Reply:
> Is there a perf improvement to this or is it just 4 characters fewer?

normally importing an api isn't a big deal, except when it's a big JS module

Reply:
I don't feel strongly, maybe 6/10. If you feel stronger, we can go with that. Otherwise, meh, let's just leave it.

Reply:
I think we can leave it like that. And yes, I ask a lot of questions, but this is the first codemod we've done, so we have to lay the foundations.